cmd/cmd_test.go

```diff
@@ -147,7 +147,8 @@ func presetConfig(fs afero.Fs) {
 	fs.Mkdir(filepath.Join(getRootPath(), ".lefthook/commit-msg"), defaultFilePermission)
 	fs.Mkdir(filepath.Join(getRootPath(), ".lefthook/pre-commit"), defaultFilePermission)
 
-	fs.MkdirAll(filepath.Join(getRootPath(), ".git", "hooks"), defaultFilePermission)
+	setGitHooksPath(".git/hooks")
```

Comment:
Why we do need to cache hooks path (if I correctly understand what is going on)?

Reply:
It's not a cache, it's more like a stub. This test is running in `cmd/` directory, and this is the way to set git hooks path to `cmd/.git/hooks`. If we use `git rev-parse --git-path hooks` here, it will return `cmd/../.git/hooks`, which is bad for tests (and breaks them actually)

Reply:
Oh, ok